### ISSUE TITLE
Recommended nginx setting overrides all subdomains

### ISSUE THREAD

**Issue by malikwirin** [nix-community/harmonia#481]
Using the recommended nginx settings
``` nix
      locations."/".extraConfig = ''
        proxy_pass http://127.0.0.1:5000;
        proxy_set_header Host $host;
        proxy_redirect http:// https://;
        proxy_http_version 1.1;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection $connection_upgrade;
      '';
```

makes it that I see ![Bildschirmfoto vom 2024-12-14 16-47-35](https://github.com/user-attachments/assets/5a4e60a6-eec1-42e6-8524-a903cfaea29b) not only when using the configured domain but when using any unconfigured domain also

**Comment by Mic92** [comment]
I don't think this is related to the configuration but to nginx itself. In my case nginx will default to my adminer instance when an unconfigured domain is hit. You can work around it by having a different vhost being the default: https://serverfault.com/questions/527156/setting-nginx-to-catch-all-unhandled-vhosts
